addons/binding/org.openhab.binding.gardena/src/main/java/org/openhab/binding/gardena/internal/GardenaSmartImpl.java

```diff
@@ -342,9 +337,10 @@ public void sendCommand(Device device, GardenaSmartCommandName commandName, Obje
         }
 
         if (command != null) {
+            stopRefreshThread();
```

Comment:
The best solution would be to delay the task at least 6 seconds. If the next execution < 6 seconds, the values are refreshed but are wrong, because the gardena cloud has not yet updated the values from the command.
So i have to cancel the refresh thread and reschedule it with a 6 seconds delay. Or do you know a better way?